frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -218,12 +218,25 @@
     await hgCommand(async () => fetch(`/api/project/hgRecover/${project.code}`));
   }
 
+  async function streamHgCommandResponse(body: ReadableStream<Uint8Array> | null): Promise<void> {
+    if (body == null) return;
+    const reader = body.getReader();
```

Comment:
rather than all this you can do this 
```ts
const decoder = new TextDecoder();
for await (const chunk of body) {
    hgCommandResponse += decoder.decode(chunk);
}
```

Reply:
Typescript says "Type 'ReadableStream<Uint8Array>' must have a '[Symbol.asyncIterator]()' method that returns an async iterator." And https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream#browser_compatibility says that that will only work on Firefox. :-(

Reply:
See https://stackoverflow.com/a/77377871 for more discussion. Going to leave as-is because the nicer `for await` solution isn't widely implemented yet.

Reply:
should just force everyone to use Firefox...